app/views/support/_time_constraint.html.erb

```diff
@@ -15,7 +15,7 @@
 
     <div class="form-group">
       <span class="form-label">
-        <%= r.label :not_before_date, "MUST NOT be published BEFORE" %>
+        <%= r.label :not_before_date, "Must not be published before" %>
```

Comment:
I can see we still need a new 'time' field for each of the above:

```
00:00
[TIME FIELD]
```

I think you're right that these changes are more involved - they'd need adding to the [`time_constraint_attributes` hash](https://github.com/alphagov/support/blob/7012da5f0f798a357b6429b9eaeb6544abb77927/app/controllers/content_change_requests_controller.rb#L23) which ultimately [populates the `WithTimeConstraint` module](https://github.com/alphagov/support/blob/b7e64c64a8e74b7221644d93bbed629ccf93b748/app/models/support/requests/with_time_constraint.rb#L13-L15). This uses the [validates_timeliness gem](https://github.com/adzap/validates_timeliness), which does appear to support times as well as dates.

Whilst these attributes do appear on the surface to also be used in [content_advice_requests_controller](https://github.com/alphagov/support/blob/7012da5f0f798a357b6429b9eaeb6544abb77927/app/controllers/content_advice_requests_controller.rb#L23) and [remove_user_requests_controller](https://github.com/alphagov/support/blob/7012da5f0f798a357b6429b9eaeb6544abb77927/app/controllers/remove_user_requests_controller.rb#L22), these are actually defined in their own templates ([1](https://github.com/alphagov/support/blob/41cec714da8967acb16a41ac93bfea4fa3368852/app/views/content_advice_requests/_request_details.html.erb#L59-L75), [2](https://github.com/alphagov/support/blob/41cec714da8967acb16a41ac93bfea4fa3368852/app/views/remove_user_requests/_request_details.html.erb#L33-L42) respectively), so we should be free to add additional fields here and not worry about any knock-on effects to other support forms.

TLDR: to add 'time' fields, you'd need to:

* add a label & input for a 'time' field next to each of the 'date' fields above
* [update the `time_constraint_attributes`](https://github.com/alphagov/support/blob/7012da5f0f798a357b6429b9eaeb6544abb77927/app/controllers/content_change_requests_controller.rb#L23)
* add some validation logic for the new fields, into [TimeConstraint](https://github.com/alphagov/support/blob/7d344be1a0bce19d830adb8e399d7b97fd7a4fcd/app/models/support/requests/time_constraint.rb) (including [corresponding tests](https://github.com/alphagov/support/blob/7012da5f0f798a357b6429b9eaeb6544abb77927/spec/models/support/requests/time_constraint_spec.rb)) - I guess as a minimum you would just validate that the time is valid ("00:00" vs "banana"), but taking it another level you could check if either of the date fields are 'today' and then check if the time field is in the future or the past. (Otherwise the time field is largely irrelevant if the date is in the future, so long as the time is valid)